app/controllers/concerns/hyrax/works_controller_behavior.rb

```diff
@@ -199,13 +199,46 @@ def contextual_path(presenter, parent_presenter)
       ::Hyrax::ContextualPath.new(presenter, parent_presenter).show
     end
 
+    # @deprecated
     def curation_concern_from_search_results
-      search_params = params
+      Deprecation.warn("We will remove '##{__method__}' in Hyrax 4.0.  " /
```

Comment:
I'm afraid that `We` might be interpreted as a royal `we` instead of a people's `we`. How about making it third-person?
```suggestion
      Deprecation.warn("'##{__method__}' will be removed in Hyrax 4.0.  " /
```

Reply:
I can certainly change it. My goal is to remove passive voice; There are implications on non-accountability.  However, it is consistent with other deprecation warnings.

Reply:
Good point.  Maybe Samvera could come up with a short style guide for deprecation text?

Reply:
is this a tech call topic?

Reply:
:+1: Added.